[2024-08-12T02:58:01.691+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-12T02:58:01.721+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather.import_weather_data_to_csv scheduled__2024-08-12T01:58:00+00:00 [queued]>
[2024-08-12T02:58:01.737+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather.import_weather_data_to_csv scheduled__2024-08-12T01:58:00+00:00 [queued]>
[2024-08-12T02:58:01.739+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-12T02:58:01.747+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): import_weather_data_to_csv> on 2024-08-12 01:58:00+00:00
[2024-08-12T02:58:01.756+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=801) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-12T02:58:01.757+0000] {standard_task_runner.py:64} INFO - Started process 803 to run task
[2024-08-12T02:58:01.757+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather', 'import_weather_data_to_csv', 'scheduled__2024-08-12T01:58:00+00:00', '--job-id', '135', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpqxtdjh4g']
[2024-08-12T02:58:01.759+0000] {standard_task_runner.py:91} INFO - Job 135: Subtask import_weather_data_to_csv
[2024-08-12T02:58:01.823+0000] {task_command.py:426} INFO - Running <TaskInstance: weather.import_weather_data_to_csv scheduled__2024-08-12T01:58:00+00:00 [running]> on host b0294fd9f5f9
[2024-08-12T02:58:01.868+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather' AIRFLOW_CTX_TASK_ID='import_weather_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-08-12T01:58:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-12T01:58:00+00:00'
[2024-08-12T02:58:01.870+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-12T02:58:01.871+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-12T02:58:01.871+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/weather_data_ingestion.py --date 2024-08-12']
[2024-08-12T02:58:01.877+0000] {subprocess.py:86} INFO - Output:
[2024-08-12T02:58:04.605+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-12T02:58:04.609+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-12T02:58:04.645+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather, task_id=import_weather_data_to_csv, run_id=scheduled__2024-08-12T01:58:00+00:00, execution_date=20240812T015800, start_date=20240812T025801, end_date=20240812T025804
[2024-08-12T02:58:04.680+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-12T02:58:04.704+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-12T02:58:04.705+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
